src/app/pages/properties/properties.component.ts

```diff
@@ -13,8 +14,8 @@ import { Title } from '@angular/platform-browser';
 })
 export class PropertiesComponent implements OnInit {
   properties: Property[] = [];
-  propertiesUrl =
-    'https://landville-backend-web-api.herokuapp.com/api/v1/properties';
+  base_url = APPCONFIG.base_url;
+  propertiesUrl =`${this.base_url}/properties/`;
```

Comment:
If the approach above is taken, then this code will change to
```suggestion
  propertiesUrl = `${environment.api_url}/properties/`;
```

Reply:
Worked on that, thanks